tests/ui/pages/desktop/collections.py

```diff
@@ -0,0 +1,29 @@
+from pypom import Region
+from selenium.webdriver.common.by import By
+
+from base import Base
+
+
+class Collections(Base):
+    """ Collections page. """
+
+    _item_locator = (By.CSS_SELECTOR, ".item")
```

Comment:
Note that single vs double quotes is only valid for strings, docstrings / comments should use double-strings at all times (this also makes a good point of using single-quotes for regular strings)